website/core/Footer.js

```diff
@@ -21,77 +21,73 @@ class Footer extends React.Component {
   render() {
     return (
       <footer className="nav-footer" id="footer">
-        {/* <section className="sitemap">
-          <a href={this.props.config.baseUrl} className="nav-home">
+        { <section className="sitemap">
+           {/* <a href={this.props.config.baseUrl} className="nav-home"> 
             {this.props.config.footerIcon && (
               <img
+                class="footer-logo"
                 src={this.props.config.baseUrl + this.props.config.footerIcon}
                 alt={this.props.config.title}
                 width="66"
                 height="58"
               />
             )}
-          </a>
+          </a> */}
           <div>
-            <h5>Docs</h5>
-            <a href={this.docUrl('doc1.html', this.props.language)}>
-              Getting Started (or other categories)
-            </a>
-            <a href={this.docUrl('doc2.html', this.props.language)}>
-              Guides (or other categories)
-            </a>
-            <a href={this.docUrl('doc3.html', this.props.language)}>
-              API Reference (or other categories)
+            <h5>Documentation</h5>
+            <a href={this.docUrl('contributing/install', this.props.language)}>
+              Install
             </a>
           </div>
           <div>
             <h5>Community</h5>
-            <a href={this.pageUrl('users.html', this.props.language)}>
-              User Showcase
-            </a>
             <a
-              href="http://stackoverflow.com/questions/tagged/"
+              href="http://stackoverflow.com/questions/tagged/fast-dna"
               target="_blank"
               rel="noreferrer noopener">
               Stack Overflow
             </a>
-            <a href="https://discordapp.com/">Project Chat</a>
             <a
-              href="https://twitter.com/"
+              href="https://discord.gg/FcSNfg4"
+              target="_blank"
+              rel="noreferrer noopener">
+              Discord
+            </a>
+            <a
+              href="https://twitter.com/FAST_DNA"
               target="_blank"
               rel="noreferrer noopener">
               Twitter
             </a>
           </div>
           <div>
             <h5>More</h5>
-            <a href={`${this.props.config.baseUrl}blog`}>Blog</a>
-            <a href="https://github.com/">GitHub</a>
+            <a
+              href="https://medium.com/fast-dna"
+              target="_blank"
+              rel="noreferrer noopener">
+              Blog
+            </a>
+            <a 
+              href="https://github.com/Microsoft/fast-dna"
+              target="_blank">
+              GitHub
+            </a>
             <a
               className="github-button"
-              href={this.props.config.repoUrl}
+              href="https://github.com/microsoft/fast-dna"
```

Comment:
I think this can still come in as props - any particular reason to hardcode this?

Reply:
It was going to the github.io url, not github. There might be a way to change the prop to get the github but I did not know it. If you do, let me know and I will make the change.